### ISSUE TITLE
AlmaLinux not correctly handled in ntopng-utils-manage-updates. Almost the same as #6491

### ISSUE THREAD

**Issue by elysch** [ntop/ntopng#6829]
**Environment**:
* OS name: AlmaLinux
* OS version: 8.6 (Sky Tiger)
* Architecture: amd64
* ntopng version/revision: ntopng-5.4.220818-18242.x86_64

**What happened**:
cron runs the ntopng-utils-manage-updates script every minute,
and produces a dpkg error every time it runs.

This happens due to line 57 execution of /usr/bin/lsb_release do not contain any of the specified terms

**How did you reproduce it?**
Wait for a minute, and an email to root is generated with the cron error.

`/usr/bin/ntopng-utils-manage-updates: line 71: dpkg: command not found`

**Debug Information**:

```
Message 4810:
From root  Thu Aug 25 19:43:06 2022
From: "(Cron Daemon)" <root>
To: root
Subject: Cron <root@appliance> /usr/bin/ntopng-utils-manage-updates -a handle-on-demand-requests
Content-Type: text/plain; charset=UTF-8
Auto-Submitted: auto-generated
Precedence: bulk
X-Cron-Env: <LANG=en_US.UTF-8>
X-Cron-Env: <SHELL=/bin/sh>
X-Cron-Env: <HOME=/root>
X-Cron-Env: <PATH=/usr/bin:/bin>
X-Cron-Env: <LOGNAME=root>
X-Cron-Env: <USER=root>
Status: R

/usr/bin/ntopng-utils-manage-updates: line 71: dpkg: command not found
```
`

Since AlmaLinux comes from RedHat, I propose the following fix that would also work in other RedHat flavored Linux servers:

```
# Detect OS
OS="DEBIAN"
if [ -x ${LSB_RELEASE} ]; then
        REDHAT=$(${LSB_RELEASE} -i | grep -i "CentOS\|RedHat\|Oracle\|Rocky\|AlmaLinux")   #**** HERE ****
        if [ ! -z "${REDHAT}" ]; then
                OS="REDHAT"
        fi
else
        if [ -f /etc/rocky-release ]; then
                OS="REDHAT"
        elif [ -f /etc/oracle-release ]; then
                OS="REDHAT"
        elif [ -f /etc/redhat-release ]; then   #**** HERE!!!! ****
                OS="REDHAT"
        fi
fi
```

**Comment by cardigliano** [comment]
Implemented in 4a000a5b69d09f56abcd3419200ee8cf2cd7358b

**Comment by elysch** [comment]
:) Thanks.
Glad to help.
